server/routes/notifications.js

```diff
@@ -0,0 +1,120 @@
+const express = require('express');
+const auth = require('../middleware/auth');
+const errorCatcher = require('../middleware/errorCatcher');
+const router = express.Router();
+
+const Notification = require('../models/Notification');
+
+// @route   GET api/notifications
+// @desc    Get all users notifications
+// @access  Private
+router.get(
+	'/',
+	auth,
+	errorCatcher(async (req, res) => {
+		const notifications = await Notification.find({
+			receiver: req.user.id,
+		}).sort({
+			date: -1,
```

Comment:
```suggestion
			date: 'desc',
```

Like we discussed in another PR👍